jobstore-persistence/pom.xml

```diff
@@ -48,16 +48,10 @@
             <scope>test</scope>
         </dependency>
         <dependency>
-            <groupId>uk.gov.justice.event-store</groupId>
-            <artifactId>test-utils-persistence</artifactId>
-            <version>${event-store.version}</version>
+            <groupId>org.apache.tomee</groupId>
+            <artifactId>apache-tomee</artifactId>
```

Comment:
Do you need all apache-tomee artefacts? Usually they are used in our IT's and we mostly use openejb-core. Please take a look at the framework.